app/src/app/voting/voting.component.ts

```diff
@@ -48,7 +52,12 @@ export class VotingComponent implements OnInit {
           })
         })
     )
-
+    this.db.object("landsraad/votingConfig/resultsShown").valueChanges().subscribe((value: boolean) => {
```

Comment:
What if no question is selected and resultsShown flag is true? Imo this will crash the app.

Reply:
Ok, I was under the impression that some question is selected always. Will test if that scenario is possible.

Reply:
Ok, so it works also for this situation:

![image](https://user-images.githubusercontent.com/35541444/211170291-4a339ba3-7922-4949-87f8-76b4aa384fdd.png)

But it is true that we do not need to run the whole data processing there, so I will see if I can return empty data object before we process the whole data.

Reply:
Right now the table, pictures and question name are just not shown when no question is selected and show results is true.

Reply:
This addition makes the code a bit more "controlled" and we do not run redundant data processing.

```
// check if we have some current question answers, if not set empty data and return
          if (Object.values(currentQuestionAnswersMap).filter(answer => answer !== undefined).length === 0) {
            this.setData(null, [])
            return
          }
```

Reply:
Changes related to this comment are in this commit: https://github.com/davidvavra/duna-valka-assassinu/pull/45/commits/963954c8814e42257b445adf921c3316cc757ca5 (for ease of review).